lib/sycamore/sycamore/transforms/extract_schema.py

```diff
@@ -219,6 +223,10 @@ def __init__(self, child: Node, schema_extractor: SchemaExtractor, **resource_ar
         super().__init__(child, f=schema_extractor.extract_schema, **resource_args)
 
 
+class OpenAIPropertyExtractor(LLMPropertyExtractor):
```

Comment:
Please add docstring and indicate this is for backwards-compatibility. Consider using the `deprecated` directive for doc generation.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated